manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.yaml

```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TencentMeeting
+PackageVersion: 3.3.2.437
```

Comment:
```suggestion
PackageVersion: 3.2.5.402
```

Reply:
I edited files in 3.2.5.402 by mistake and reverted, but I forgot to edit files for 3.3.2.437. Now it's fixed.